modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2RoleClaimTestCase.java

```diff
@@ -172,112 +186,51 @@ public void testSendAuthorizedPostAfterRoleUpdate() throws Exception {
                 "Id token must not contain role claim which is not configured for the requested scope.");
     }
 
-    private ClaimValue[] getUserClaims() {
+    private UserObject getUserCreationInfo() {
+        UserObject userInfo = new UserObject();
```

Comment:
Shall we fix the formatting issue by adding the missing new line?

Reply:
Fixed with https://github.com/wso2/product-is/pull/20096